src/plugins.py

```diff
@@ -50,9 +52,15 @@ def loadPlugin(self, i):
 		except: author = None
 		try: website = plugin.WEBSITE
 		except: website = None
-		if id in self.wrapper.storage["disabled_plugins"] or plugin.DISABLED:
```

Comment:
and by the way,  this part was my fault (not assigning `plugin.DISABLED` to `disabled` and as a result making this "if" statement test against plugin.Disabled).  So this part was a correction to an earlier PR of mine.